mmcv/runner/hooks/logger/wandb.py

```diff
@@ -45,8 +45,20 @@ def log(self, runner):
         tags = self.get_loggable_tags(runner)
         if tags:
```

Comment:
@joihn @ZwwWayne @MengzhangLI should be:

```python
tags = self.get_loggable_tags(runner)
if tags:
    tags['global_step'] = self.get_iter(runner)
    self.wandb.log(tags, commit=self.commit)
```

check open-mmlab/mmsegmentation#1068.